### ISSUE TITLE
add manifest declaration for Win10 DPI support

### ISSUE THREAD

**Issue by sean-e** [sean-e/mTroll#2]
https://blogs.windows.com/buildingapps/2017/05/19/improving-high-dpi-experience-gdi-based-desktop-apps/
```xml
<asmv3:application>
	<asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2017/WindowsSettings">
		<gdiScaling>true</gdiScaling>
	</asmv3:windowsSettings>
</asmv3:application>
```

**Comment by sean-e** [comment]
https://msdn.microsoft.com/en-us/library/windows/desktop/aa374191%28v=vs.85%29.aspx?f=255&MSPPError=-2147217396

**Comment by sean-e** [comment]
implemented via Qt API call in 12dd45150d62a7b2d70a007cd0ce2a070bccd5cc rather than via manifest
